yarn-project/ethereum/src/deploy_l1_contracts.ts

```diff
@@ -222,6 +223,19 @@ export type L1Clients = {
   walletClient: WalletClient<HttpTransport, Chain, Account>;
 };
 
+export const deployerAbi = [
+  {
+    inputs: [
+      { internalType: 'bytes32', name: 'salt', type: 'bytes32' },
+      { internalType: 'bytes', name: 'init_code', type: 'bytes' },
+    ],
+    name: 'deploy',
+    outputs: [{ internalType: 'address', name: '', type: 'address' }],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+];
```

Comment:
Are we using this abi at all? IIRC we had found that the deployer was not abi-compliant, right?